cowprotocol/cowswap
# Update Goerli cow and vcow address and merkle repo

This updates COW and vCOW addresses for Goerli and also sets new branch (main) and repo for [cow-merkle-drop](https://github.com/cowprotocol/cow-merkle-drop)
The end result is that vCow claiming is available on Goerli
![Screenshot from 2022-10-28 12-55-21](https://user-images.githubusercontent.com/34926005/198573230-4182cbf3-aa5a-42a5-a013-db05fbc68b93.png)

# To test
1. There is a message on slack posted by Federico in `#token` channel, with the list of addresses that have claims on Goerli or you can use one of the following addresses 
`0x005c2eD5604BAeB3Fd8fb2DE710b4481aE4A847e`
`0x00A651705B9048614d91BBB81C9244ec65Da5859`
`0x04B6C0F0ae6b49c207136D2ff1A57704BA313C9F`
`0x059E927e96079AffCC63583399641d1Da33e9Dd2`
`0x05A1b71BF498d4A3930BA1f940E2D58C2150b399`
3. Take one address and find the PK in the excel claims file
4. Log in with that account and go to account overview page
5. You should see Card where you can convert your vCow to cow
